content/docs/reference/routes/set-request-headers.mdx

```diff
@@ -52,13 +54,15 @@ See Kubernetes [Ingress](/docs/deploy/k8s/ingress#annotations) for more informat
 </TabItem>
 </Tabs>
 
-### Pass ID and Access tokens
+### Pass dynamic values in headers
 
 | **Option** | **Value** | **Type** | **Usage** |
 | :-- | :-- | :-- | :-- |
 | `x-pomerium-downstream-id-token` | `$pomerium.id_token` | Map of key-value pairs | **optional** |
 | `x-pomerium-downstream-access-token` | `$pomerium.access_token` | Map of key-value pairs | **optional** |
 
+**Note:** Token values must use the `$pomerium.<token>` syntax.
+
```

Comment:
This table (and the one above it) doesn't make sense to me. I don't think the "Option" heading is accurate, and I think the "Map of key-value pairs" belongs in the previous table, under the "Type" heading.

If you like I'd be happy to update this section.

Reply:
@kenjenkins the first table lists the configuration key, data type, and whether the setting is required/optional. It's the original setting to be used with static header values (I reformatted that data into a table, but before it was an unordered list). 

The second table addresses the new functionality that allows you to pass in dynamic values into headers for the tokens. We've used `map of key-value pairs` in other config settings, so I've applied it here.  As for labeling this table **Options**, I'm open to calling it something else. 

What about it doesn't make sense? Asking because we've applied the same formatting to pretty much every reference entry so far.

I'm hesitant to hand it off entirely just because I want to maintain consistency with our reference entries, but I'd like to see what your suggestion is.

Reply:
Here's a concrete proposal, hopefully that can help clarify my concerns: https://github.com/pomerium/documentation/commit/7a54d1d83f91a9e86f79304a0f451e9452e0ff27

Specifically:
 - the **Type** column in the first table does not make sense to me — as I understand it this option is not a `string`, it is a mapping (both before and after the change to support the new `$pomerium.<token>` values)
 - the available token options (`$pomerium.id_token` and `$pomerium.access_token`) are not themselves key-value pairs; rather they can be used as part of the value
 - it doesn't make sense to me to include example header names (`x-pomerium-downstream-id-token`) in the table, as these are up to the user to decide (right?)

Does that make sense?

Reply:
Hey Ken, so:
- That's fine, we can update that to `map of key-value pairs` (or the suggestion in your proposal above)
- As for the token options (`$pomerium.<token>`), I'm aware those aren't key-value pairs, the confusion lies in the point below
- I was under the assumption that those header values (what I thought of as the `key`) weren't customizable, but hardcoded values (`x-pomerium-downstream-id-token`), hence the reason I included them in a table as an Option, and the value as `$pomerium.<token>`. 

Thanks for taking the time to check this out to make sure it's technically accurate. What I'll do is update this PR with your suggestions (early tomorrow). I'll ping you on Slack so you can review.